librz/il/il_validate.c

```diff
@@ -136,6 +137,8 @@ static bool local_context_init(LocalContext *ctx, const RzILValidateGlobalContex
 static void local_context_fini(LocalContext *ctx) {
 	ht_pp_free(ctx->local_vars_known);
 	ht_pp_free(ctx->local_vars_available);
+	ctx->local_vars_known = NULL;
+	ctx->local_vars_available = NULL;
```

Comment:
This should not be necessary. Did you encounter any place where there was a double free?

Reply:
oh i just set them to NULL for getting a clean view of memory when I was debugging. not be necessary one